media/js/base/translation-bar-init.es6.js

```diff
@@ -0,0 +1,9 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+import translationBar from './translation-bar.es6.js';
+
+window.Mozilla.Utils.onDocumentReady(translationBar.init());
```

Comment:
There's some JS functions in here like `fetch` and `Map` that aren't always widely supported on older browsers, and Babel might not compile away. I think for this feature it's fine to keep using newer methods, but we should try and avoid throwing errors in older browsers if possible.

Either using a feature detect or (if that's not so easy) a try/catch might be worthwhile to prevent a bunch of noise in Sentry.

We should probably also give this a good bit of testing on demo once this PR is ready.